test/functional/p2p_orphan_handling.py

```
@@ -0,0 +1,377 @@
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import time
+
+from test_framework.messages import (
+    CInv,
+    MSG_TX,
+    msg_getdata,
+    msg_inv,
+    msg_notfound,
+    msg_tx,
+    tx_from_hex,
```

Ruff (0.12.2)

`test_framework.messages.tx_from_hex` imported but unused

Remove unused import: `test_framework.messages.tx_from_hex`

(F401)